src/FillArrays.jl

```diff
@@ -207,9 +207,11 @@ function +(a::Fill{T, 1}, b::AbstractRange) where {T}
     return a.value .+ b
 end
 +(a::AbstractRange, b::AbstractFill) = b + a
++(a::AbstractFill{<:Any,2}, b::UniformScaling) = Array(a) + b
 
 -(a::AbstractFill, b::AbstractRange) = a + (-b)
 -(a::AbstractRange, b::AbstractFill) = a + (-b)
+-(a::UniformScaling, b::AbstractFill{<:Any,2}) = a + (-b)
```

Comment:
Similarly here, `-(::AbstractFill{<:Any,2}, ::UniformScaling)` falls back to `+(::AbstractFill{<:Any,2}, ::UniformScaling)`: https://github.com/JuliaLang/julia/blob/e3b681c6872d87d74db63dd7ec3da2beee8e3ae4/stdlib/LinearAlgebra/src/uniformscaling.jl#L159